docs/visualize/tilemap.asciidoc

```diff
@@ -1,105 +1,72 @@
 [[tilemap]]
-== Coordinate Maps
+== Coordinate map
 
-A coordinate map displays a geographic area overlaid with circles keyed to the data determined by the buckets you specify.
+Coordinate maps display geographic areas overlaid with circles keyed to the data determined by the buckets you specify. To use coordinate maps, you plot latitude and longitude coordinates on a map.
 
-Kibana’s out-of-the-box settings do not show a coordinate map in the New Visualization menu. Use <<maps>> instead, which offers more functionality and is easier to use.
-If you want to create new coordinate map visualizations, set `xpack.maps.showMapVisualizationTypes` to `true`.
+Coordinate maps have been replaced with <<maps>>, which offers more functionality and is easier to use.
 
-Kibana uses the https://www.elastic.co/elastic-maps-service[Elastic Maps Service]
-to display map tiles. To use other tile service providers, configure the <<tilemap-settings,tilemap settings>>
+If you still want to create coordinate maps in Visualize:
+
+. Set `xpack.maps.showMapVisualizationTypes` to `true`.
+
+. To display map tiles, {kib} uses the https://www.elastic.co/elastic-maps-service[Elastic Maps Service].
```

Comment:
The first sentence in this bullet point surprised me. I expected it to be a step that I should perform.